app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -98,17 +98,8 @@ public void onCreate(Bundle savedInstanceState) {
         binding = ActivityLoginBinding.inflate(getLayoutInflater());
         setContentView(binding.getRoot());
 
-
-        //Get Extra message string from the intent and show Toast
-        String message = getIntent().getStringExtra("loginMessage");
-        String username = getIntent().getStringExtra("username");
-        if(message!=null) {
-            showMessage(message, R.color.secondaryDarkColor);
-        }
-        if(username!=null) {
-            binding.loginUsername.setText(username);
-        }
-
+        String message = getIntent().getStringExtra(getString(R.string.login_message));
```

Comment:
I think these two should be constants here, rather than resource strings (e.g. because we don't want to translate these etc.)